projects/RabbitMQ.Client/Impl/Channel.PublisherConfirms.cs

```diff
@@ -249,7 +252,8 @@ private void HandleReturn(BasicReturnEventArgs basicReturnEvent)
                     }
                 }
 
-                HandleNack(publishSequenceNumber, multiple: false, isReturn: true);
+                HandleNack(publishSequenceNumber, multiple: false, isReturn: true,
```

Comment:
Would it be feasible to also pass `basicReturnEvent.ReplyCode` and `basicReturnEvent.ReplyText` here? Then we could have these two also in the `PublishReturnException` to provide additional context.

Reply:
Done, thanks for noticing that!